index.html

```diff
@@ -16,7 +16,7 @@
         <div class="form-group">
           <input type="text" name="book[id]" id="book-id" placeholder="Resource ID Here">
         </div>
-        <button type="submit" class="btn btn-default">Get all Books</button>
+        <button type="submit" class="btn btn-default">Submit</button>
```

Comment:
So, I think the point here is to add to the app's functionality, not overwrite what we did before. This should be done as a new form, so that they now have both index and show on their app.